### DISCUSSION TITLE
error loading mods

### DISCUSSION THREAD

**Discussion by THenryInGH** [Meloong-Git/PCL#3545] [提问]
各位大神急救，在运行游戏时遇到错误信息。
[错误报告-20-03-2024_20.27.40.zip](https://github.com/Hex-Dragon/PCL2/files/14667061/-20-03-2024_20.27.40.zip)

**Comment by allMagicNB** [comment]
Java 擦腚纸的问题？

**Reply by THenryInGH** [reply]
具体有什么解决方法吗？

**Reply by wuliaodexiaoluo** [reply]
```Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.UnsatisfiedLinkError: D:\Henry\Minecraft????\?????\.minecraft\versions\1.20.4-Forge_49.0.31-OptiFine_I7\1.20.4-Forge_49.0.31-OptiFine_I7-natives\jna9137023719633516996.dll: Can't find dependent libraries [in thread "main"]```
看起来像是有零宽连字或者其他 Unicode 控制符存在于 Path 中

**Reply by THenryInGH** [reply]
有更改了但是还是会出现optifine那边的jar was not found

**Reply by wuliaodexiaoluo** [reply]
Forge 的 Bug，这个真的解决不了

**Reply by THenryInGH** [reply]
ok谢谢！我去下载旧一些的版本就可以了
